samcli/lib/utils/osutils.py

```diff
@@ -0,0 +1,82 @@
+"""
+Common OS utilities
+"""
+
+import sys
+import os
+import shutil
+import tempfile
+
+from contextlib import contextmanager
+
+
+@contextmanager
+def mkdir_temp(mode=0o755):
```

Comment:
do we want to make a comment, on why we chose these permissions?

Reply:
Will do.